pmd-java/src/main/resources/rulesets/java/codesize.xml

```diff
@@ -448,6 +451,50 @@ public class Foo extends Bar {
    </example>
 </rule>
 
+    <rule name="NcssCount"
+          message="The {0} ''{1}'' has a NCSS line count of {2}."
+          since="3.9"
```

Comment:
FYI - I've changed this attribute to be "since=6.0", as this rule is actually new.
We also have the "tradition" to add new rules into special internal rulesets, e.g. for 5.8.0 it was [580.xml](https://github.com/pmd/pmd/blob/master/pmd-core/src/main/resources/rulesets/releases/580.xml). I'll create a new file 600.xml and reference this rule then.